src/Button/ButtonBase.tsx

```diff
@@ -21,6 +21,7 @@ const ButtonBase = forwardRef(
       size = 'medium',
       alignContent = 'center',
       block = false,
+      inactive,
```

Comment:
Curious, what if we just continue to use `disabled` as a prop but change it to use `aria-disabled` instead of `disabled` across the board? 

Is this primarily used for the loading state? So it looked disabled but it can be focused.. can it also still be clicked and follow through with the on click action?

Reply:
I think there are still rare cases where we want a real `disabled` button.

The loading state is one way we can use this pattern, but it's mostly to support "degraded" buttons: https://primer.style/design/ui-patterns/degraded-experiences#indicating-a-button-is-non-functional-without-disabling-it

Reply:
> Curious, what if we just continue to use disabled as a prop but change it to use aria-disabled instead of disabled across the board?

I like the general idea of that, but it should come with handling click events with a useful message, which I'm not confident would happen :(

> I think there are still rare cases where we want a real disabled button.

Also fair!